lib/joken/token.ex

```diff
@@ -71,7 +71,7 @@ defmodule Joken.Token do
         {:ok, data} = get_data(token, joken_config)
 
         results = Enum.into(claims, []) |> Enum.map(fn(claim) ->
-          joken_config.validate_claim(claim, data)
+          joken_config.validate_claim(claim, data, options)
```

Comment:
I think here we would need to get all claim keys from the claims, but also from the options yes? Something like:

    Dict.keys(claims) ++ Dict.keys(options) 
    |> Enum.uniq
    |> Enum.map.....